lms/static/js/fixtures/dashboard/dashboard.html

```diff
@@ -20,3 +20,92 @@ <h3 class="course-title">
+                                    </li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <footer class="wrapper-messages-primary">
+            <ul class="messages-list">
+                <div class="message message-upsell has-actions is-shown">
+                    <div class="wrapper-extended">
+                        <p class="message-copy" align="justify">
+                            <b class="message-copy-bold">Pursue a Certificate of Achievement to highlight the knowledge and skills you gain in this course.</b>
+                            <br>
+                            It's official. It's easily shareable. It's a proven motivator to complete the course.
+                            <br>
+                            <a href="https://www.edx.org/verified-certificate" class="verified-info" data-course-key="course-v1:DelftX+CTB3365DWx+1T2016">Learn more about the verified Certificate of Achievement</a>.
+                        </p>
+                        <div class="action-upgrade-container">
+                            <a class="action action-upgrade" href="/verify_student/upgrade/course-v1:DelftX+CTB3365DWx+1T2016/" data-course-id="course-v1:DelftX+CTB3365DWx+1T2016" data-user="Anon">
+                                <i class="action-upgrade-icon"></i>
+                                <span class="wrapper-copy">
+                                    <span class="copy" id="upgrade-to-verified">Upgrade to Verified</span>
+                                </span>
+                            </a>
+                        </div>
+                    </div>
+                </div>          
+                <div class="message message-status is-shown credit-message">
+                    <div class="xseries-action">
+                        <div class="message-copy xseries-msg">
+                            <p><b class="message-copy-bold">XSeries Program: Interested in more courses in this subject?</b></p>
+                            <p></p>
+                            <p class="message-copy">
+                                This course is 1 of 3 courses in the <a href="https://www.edx.org/xseries/water-management">Water Management</a> XSeries.
+                            </p>
+                        </div>
+                        
+                        <a class="btn xseries-border-btn" href="https://www.edx.org/xseries/water-management" target="_blank"
+                           data-program-id="xseries007">
+                            <i class="action-xseries-icon"></i>
+                            <span>View XSeries Details</span>
+                        </a>
+                    </div>
+                </div>
+            </ul>
+        </footer>
+    </article>
+</div>
```

Comment:
Yeah, there is a function getting data from the DOM tree so it has to be in place